autogen.sh

```diff
@@ -11,3 +11,5 @@ autoreconf --verbose --force --install || exit $?
 
 cd "$cwd"
 test -z "$NOCONFIGURE" && "$srcdir/configure" $@
+
+gtkdocize || exit 1
```

Comment:
This also assumes the user has GTK-Doc on their system, which might not be the case. There should be a check for the `gtkdocize` binary before calling it. Otherwise, this script will always fail for people without GTK-Doc.